src/jit/lclvars.cpp

```diff
@@ -3861,7 +3863,11 @@ void Compiler::lvaAssignVirtualFrameOffsetsToArgs()
     {
         noway_assert(lclNum == info.compThisArg);
 #ifndef _TARGET_X86_
```

Comment:
NIT: Would read better as:
#ifdef UNIX
Call()
#else
Call(foo)
#endif

in all places.